data/json/monsters/feral_humans.json

```
@@ -58,6 +58,65 @@
       "DROPS_AMMO"
     ]
   },
+  {
+    "id": "mon_feral_sapien_spear",
+    "type": "MONSTER",
+    "name": { "str": "feral sapien" },
+    "description": "In this feral's eyes is more intelligence than in most of their kind.  They appear to be holding a spear.",
```

```suggestion
    "description": "In this feral's eyes is more intelligence than in those of most of their kind.  They appear to be holding a spear.",
```
It's clearer, but wordier... hrm.